kustomize/commands/edit/add/addreplacement.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2023 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package add
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kustomize/api/konfig"
+	"sigs.k8s.io/kustomize/api/types"
+	"sigs.k8s.io/kustomize/kustomize/v5/commands/internal/kustfile"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+type addReplacementOptions struct {
+	Replacement types.ReplacementField
+}
+
+func newCmdAddReplacement(fSys filesys.FileSystem) *cobra.Command {
+	var o addReplacementOptions
+	/*o.Replacement.Source = &types.SourceSelector{}
+	o.Replacement.Targets = append(o.Replacement.Targets, &types.TargetSelector{})*/
```

Comment:
Could we remove this commented block entirely?
```suggestion
```

Reply:
removed